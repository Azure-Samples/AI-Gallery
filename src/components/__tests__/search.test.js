import React from 'react';
import renderer from 'react-test-renderer'
import Search from './../search';


test('renders a snapshot of Search', () => {
    const tree = renderer.create(<Search/>).toJSON();
    expect(tree).toMatchSnapshot();
})  