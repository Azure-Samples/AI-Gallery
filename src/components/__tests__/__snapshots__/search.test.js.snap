// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of Search 1`] = `
<div>
  <div
    class="content-container"
  >
    <input
      className="button btnSearch searchbtn noselect"
      id="btnSearch"
      onClick={[Function]}
      type="button"
      value="Search"
    />
    <div
      class="searchbar-container"
    >
      <input
        class="searchbar"
        id="keyword"
        name="keyword"
        onChange={[Function]}
        type="text"
        value=""
        width="100%"
      />
    </div>
  </div>
  <div
    className="gh-list"
  >
    <h3>
      Current Topic: '
      
      '
    </h3>
    <ul
      classname="gh-frame-container"
    />
  </div>
</div>
`;
